tests/Issues/AnnotationImport/Fixture/skip_with_namespace_not_exists.php.inc

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+/**
+ * @Doctrine\ORM\Mapping\Table("Table_Name")
```

Comment:
for namespaced class, eg, has `namespace App;`, if it going to be imported, it should be with `App` prefix: `App\Doctrine\ORM\Mapping\Table`, so I think it should be skipped instead.

Reply:
fixture removed for now so this PR only handle based on reported issue https://github.com/rectorphp/rector/issues/7051 for non-namespaced file.